This patch enables higher gain from rtlsdrs, both in manual and in AGC mode

diff --git a/src/tuner_r82xx.c b/src/tuner_r82xx.c
index f620238..ce77a2d 100644
--- a/src/tuner_r82xx.c
+++ b/src/tuner_r82xx.c
@@ -1027,8 +1027,8 @@ int r82xx_set_gain(struct r82xx_priv *priv, int set_manual_gain, int gain)
 		if (rc < 0)
 			return rc;
 
-		/* set fixed VGA gain for now (16.3 dB) */
-		rc = r82xx_write_reg_mask(priv, 0x0c, 0x08, 0x9f);
+		/* set MAX gain */
+		rc = r82xx_write_reg_mask(priv, 0x0c, 0x0f, 0x9f);
 		if (rc < 0)
 			return rc;
 
@@ -1064,8 +1064,8 @@ int r82xx_set_gain(struct r82xx_priv *priv, int set_manual_gain, int gain)
 		if (rc < 0)
 			return rc;
 
-		/* set fixed VGA gain for now (26.5 dB) */
-		rc = r82xx_write_reg_mask(priv, 0x0c, 0x0b, 0x9f);
+		/* set MAX gain */
+		rc = r82xx_write_reg_mask(priv, 0x0c, 0x0f, 0x9f);
 		if (rc < 0)
 			return rc;
 	}
